backend_addon/{{ cookiecutter.__folder_name }}/.github/workflows/ci.yml

```diff
@@ -0,0 +1,128 @@
+name: {{ cookiecutter.python_package_name }} CI
+
+on:
+  push:
+
+env:
+  PYTHON_VERSION: "{{ cookiecutter.__gha_python_version }}"
+  PLONE_VERSION: "{{ cookiecutter.__gha_plone_version }}"
+
+jobs:
+
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+
+        - name: Checkout
+          uses: actions/checkout@{{ cookiecutter.__gha_version_checkout }}
+
+        - name: Install the latest version of uv
+          uses: astral-sh/setup-uv@{{ cookiecutter.__gha_version_setup_uv }}
+          with:
+            python-version: {{ "${{ env.PYTHON_VERSION }}" }}
```

Comment:
Should this enable caching? https://github.com/astral-sh/setup-uv?tab=readme-ov-file#enable-caching